BBBadgeBarButtonItem/BBBadgeBarButtonItem.m

```diff
@@ -142,6 +141,7 @@ - (void)setBadgeValue:(NSString *)badgeValue
         self.badge.backgroundColor      = self.badgeBGColor;
         self.badge.font                 = self.badgeFont;
         self.badge.textAlignment        = NSTextAlignmentCenter;
+        self.badge.layer.masksToBounds  = YES;
 
```

Comment:
instead of copying the files or changing the code in the pod folder each time you do a pod install/update .... you can just update your pod file to point to this commit with the fix
```
pod 'BBBadgeBarButtonItem', :git => 'https://github.com/dkalachov/BBBadgeBarButtonItem.git', :commit => '3a673aedb53718bf67ad8f5a6a61b8966845f9f7'
```